### ISSUE TITLE
Long lag while loading plugin

### ISSUE THREAD

**Issue by woj-tek** [fernandokosh/redmine_time_tracker#49]
Hi,
we've installed your plugin (0.4) on our Redmine 1.1.1.stable but there is a problem. On every page there is noticeable lag between page loading (everything is visible) and time when the plugin shows (about 30 secons) when there seems like nothing is happening. If I stop loading the page right after it loads (I mean the main content, while the plugin still tries to load) then it appears right away.

Any Idea what could be the cause or how could it be debugged?

It works like that on: Opera 11.10, Internet Explorer 9.0, Firefox 4.0. Latest Chrome (in my case ChromePlus) seems to work OK though...

**Comment by delaitre** [comment]
This is weird! I've never noticed such problem (never tested on 1.1.1
though).
If I understand correctly, the whole redmine page is displayed correctly
except the plugin which is not displayed at all. Then, few seconds later,
the plugin is displayed. Am I right?

I have no idea where it can come from... Maybe a strange Redmine change or
regression.

**Comment by woj-tek** [comment]
Yes, that is exactly the case and it only occurs in mentioned browsers (which makes matters even weirder). Which Redmine version would you recommend/have you tested tracker with?

**Comment by delaitre** [comment]
Well, I usually take the last stable redmine's version and ran it in a virtual machine when I develop a new feature for the plugin.
However, I'm still running an old 0.9.2 redmine at my work... nobody as time to update it... (yeah, this mean I cannot use the last version of my own plugin!)

In other words, I'm not the right person to ask for the better version to run the plugin. I think 0.4 was developed on redmine 1.1.0 without problem though.
